doc/ref_model/chapters/chapter09.md

```diff
@@ -243,3 +247,82 @@ auto.cicd.014 | The CD pipeline must be able to automatically promote software f
 <a name="9.5.3.2"></a>
 #### 9.5.3.2. Tenant Networking Automation
 
+
+<a name="9.6"></a>
+## 9.6. Telemetry and Observability
+
+Operating complex distributed systems, such as a Telco network, is a demanding and challenging task, that is continuously being increased as the network complexity and the production excellence requirements grow. There are multiple reasons why it is so, but they originate in the nature of the system concept. To reach the ability of providing Telco services, a complex system is decomposed into multiple different functional blocks, called network functions. Internal communication between the diverse network functions of a distributed system is based on message exchange. To formalize this communication, clearly defined interfaces are introduced, and protocols designed. Even though the architecture of a Telco network is systematically formalized on the worldwide level, heterogeneity of services, functions, interfaces, and protocols cannot be avoided. By adding the multi-vendor approach in implementation of Telco networks, the outcome is indeed a system with remarkably high level of complexity which requires significant efforts for managing and operating it.
```

Comment:
@sukhdevkapur This first paragraph concerns the all chapter. I have the feeling that it arrives too late in the chapter. It could be moved to the introduction, as section 9.2 "Configuration and LCM" talks about observed state and section 9.3 "Assurance" is linked to monitoring

Reply:
@karinesevilla - Are you suggesting that we move section 9.6 relatively higher in the chapter or are you suggesting moving just one paragraph. If latter, then it will probably break the continuity of section 9.6

Reply:
@sukhdevkapur My suggestion was to move the first paragraph only. But if it breaks the text continuity, let it as it is for this release, we can rebalance after Elbrus

Reply:
@karinesevilla - Yes. Lets keep it together for now. We can always revisit it later